framework/components/ecl-links/ecl-links.twig

```diff
@@ -1,5 +1,25 @@
-<div>Lorem ipsum, <a href="#" class="ecl-link">in-line link</a> consectetur adipiscing elit</div>
-<div>Lorem ipsum, <a href="#" class="ecl-link ecl-link--external">external link</a> consectetur adipiscing elit</div>
-<div><a href="#" class="ecl-link ecl-link--standalone">Stand-alone link</a></div>
-<div><a href="#" class="ecl-link ecl-link--inverted ecl-u-bg-shade">Inverted link</a></div>
-<div class="ecl-clearfix" style="width: 50%"><a href="#" class="ecl-link ecl-link--all">'View all' link</a></div>
+{#
+  - variant: can be '','external','standalone', 'inverted', 'all', 'more' (default: '')
+  - href: target of the link (default: '')
+  - label: label of the link (default: '')
+#}
+
+{# Internal properties #}
+
+{% set _cssClass = 'ecl-link' ~ (variant != '' ? ' ecl-link--' ~ variant: '') %}
```

Comment:
The [twig coding standard](https://twig.symfony.com/doc/2.x/coding_standards.html)) are not respected